Hollowborn/HollowbornChaosEnvoy/CoreHollowbornChaosEnvoy.cs

```diff
@@ -84,8 +84,8 @@ public void StirringDiscord(bool getAll = true)
         while (!Bot.ShouldExit && !Core.CheckInventory(rewards, any: !getAll))
         {
             Core.EquipClass(ClassType.Solo);
-            Core.HuntMonster("lagunabeach", "Heart of Chaos", "Chaos Pirate Crew", isTemp: false);
-            Core.HuntMonster("backroom", "Book Wyrm", "Maledictus Magum", isTemp: false);
+            Core.HuntMonster("lagunabeach", "Heart of Chaos", "Chaos Pirate Crew", isTemp: false, publicRoom: true);
```

Comment:
[not an isssue] as much as you think thisll help it'll only be for those that have pubic on difficult set to enabled. good one though i guess

Reply:
the boss is not too hard-hitting but has a high HP of around 500k and the drop rate is very very low~